### PR TITLE
Translate JavaScript chapter to Spanish

### PR DESCRIPTION
**Pull request by c-torres**
Progress on #498

### CODE REVIEW

**Comment by rviscomi** [comment]
I'll defer to @taytus and @JMPerez for the content review. Feel free to merge when approved.

File: src/content/es/2019/javascript.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+La especificación del lenguaje en sí, junto con muchas bibliotecas y marcos creados por la comunidad utilizados por desarrolladores de todo el mundo, ha cambiado y evolucionado desde que se creó el lenguaje en 1995. Las implementaciones e intérpretes de JavaScript también han seguido progresando, haciendo que el lenguaje sea utilizable en muchos entornos, no solo navegadores web.
+
+[HTTP Archive](https://httparchive.org/) explora [millones de páginas](https://httparchive.org/reports/state-of-the-web#numUrls) todos los meses y los ejecuta a través de una instancia privada de [WebPageTest](https://webpagetest.org/) para almacenar información clave de cada página. (Puedes aprender más sobre esto en nuestra [metodología](./methodology)). En el context de JavaScript, HTTP Archive proporciona información extensa sobre el uso del idioma para toda la web. Este capítulo consolida y analiza muchas de estas tendencias.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
[HTTP Archive](https://httparchive.org/) explora [millones de páginas](https://httparchive.org/reports/state-of-the-web#numUrls) todos los meses y los ejecuta a través de una instancia privada de [WebPageTest](https://webpagetest.org/) para almacenar información clave de cada página. (Puedes aprender más sobre esto en nuestra [metodología](./methodology)). En el context de JavaScript, HTTP Archive proporciona información extensa sobre el uso del lenguaje en toda la web. Este capítulo consolida y analiza muchas de estas tendencias.
```

File: src/content/es/2019/javascript.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+## ¿Cuánto JavaScript usamos?
+
+JavaScript es el recurso más costoso que enviamos a los navegadores; ya que tiene que ser descargado, analizado, compilado y finalmente ejecutado. Aunque los navegadores han disminuido significativamente el tiempo que lleva analizar y compilar scripts, [la descarga y la ejecución se han convertido en las etapas más caras](https://v8.dev/blog/cost-of-javascript-2019) cuando JavaScript es procesado por una página web.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
JavaScript es el recurso más costoso que enviamos a los navegadores, ya que tiene que ser descargado, analizado, compilado y finalmente ejecutado. Aunque los navegadores han disminuido significativamente el tiempo que lleva analizar y compilar scripts, [la descarga y la ejecución se han convertido en las etapas más costosas](https://v8.dev/blog/cost-of-javascript-2019) cuando JavaScript es procesado por una página web.
```

File: src/content/es/2019/javascript.md [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+
+Enviar paquetes de JavaScript más pequeños al navegador es la mejor manera de reducir los tiempos de descarga y, a su vez, mejorar el rendimiento de la página.**Pero, ¿cuánto JavaScript utilizamos realmente?**
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig1.png">
+      <img src="/static/images/2019/01_JavaScript/fig1.png" alt="Figura 1. Distribución de bytes de JavaScript por página." aria-labelledby="fig1-caption" aria-describedby="fig1-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1974602890&format=interactive">
+   </a>
+	<div id="fig1-description" class="visually-hidden">Gráfico de barras que muestra que 70 bytes de JavaScript se usa en el percentil p10, 174 bytes para p25, 373 bytes para p50, 693 bytes para p75 y 1,093 bytes para p90</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig1-description" class="visually-hidden">Gráfico de barras que muestra que 70 bytes de JavaScript se usa en el percentil p10, 174 bytes para p25, 373 bytes para p50, 693 bytes para p75 y 1.093 bytes para p90</div>
```

File: src/content/es/2019/javascript.md [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig2.png">
+      <img src="/static/images/2019/01_JavaScript/fig2.png" alt="Figura 2. Distribución de JavaScript por página por dispositivo." aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1914565673&format=interactive">
+   </a>
+	<div id="fig2-description" class="visually-hidden">Gráfico de barras que muestra 76 bytes / 65 bytes de JavaScript se usa en el percentil p10 en computadoras de escritorio y dispositivos móviles respectivamente, 186/164 bytes para p25, 391/359 bytes para p50, 721/668 bytes para p75 y 1,131 / 1,060 bytes para p90 .</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig2-description" class="visually-hidden">Gráfico de barras que muestra 76 bytes / 65 bytes de JavaScript se usa en el percentil p10 en computadoras de escritorio y dispositivos móviles respectivamente, 186/164 bytes para p25, 391/359 bytes para p50, 721/668 bytes para p75 y 1.131 / 1.060 bytes para p90 .</div>
```

File: src/content/es/2019/javascript.md [hunk 5]
```diff
@@ -0,0 +1,439 @@
...
+	<figcaption id="fig2-caption">Figura 2. Distribución de JavaScript por página por dispositivo.</figcaption>
+</figure>
+
+En cada percentil, estamos enviando un poco más de JavaScript a los dispositivos de escritorio que a los dispositivos móviles.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
En cada percentil estamos enviando un poco más de JavaScript a los dispositivos de escritorio que a los dispositivos móviles.
```

File: src/content/es/2019/javascript.md [hunk 6]
```diff
@@ -0,0 +1,439 @@
...
+
+### Tiempo de procesamiento
+
+Después de ser analizado y compilado, el JavaScript cargado por el navegador debe procesarse (o ejecutarse) antes de poder ser utilizado. Los dispositivos varían y su potencia informática puede afectar significativamente la rapidez con que se puede procesar JavaScript en una página. ¿Cuáles son los tiempos de procesamiento actuales en la web?
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Después de ser analizado y compilado, el JavaScript cargado por el navegador debe procesarse (o ejecutarse) antes de poder ser utilizado. Los dispositivos varían y su potencia de cálculo puede afectar significativamente la rapidez con que se puede procesar JavaScript en una página. ¿Cuáles son los tiempos de procesamiento actuales en la web?
```

File: src/content/es/2019/javascript.md [hunk 7]
```diff
@@ -0,0 +1,439 @@
...
+   </a>
+	<div id="fig3-description" class="visually-hidden">Gráfico de barras que muestra 141 ms / 377 ms de tiempo de procesamiento se utiliza en el percentil p10 en computadoras de escritorio y dispositivos móviles respectivamente, 352/988 ms para p25, 849 / 2.437 ms para p50, 1.850 / 5.518 ms para p75 y 3.543 / 10.735 ms para p90.</div>
+	<figcaption id="fig3-caption">Figura 3. Tiempos de procesamiento de subprocesos principales según V8 por dispositivo.</figcaption>
+</figure>
+
+En cada percentil, los tiempos de procesamiento son más largos para las páginas web móviles que para las computadoras de escritorio. La mediana del tiempo total de subprocesos principales en el escritorio es de 849 ms, mientras que el móvil está en un número mayor: 2.436 ms.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
En cada percentil los tiempos de procesamiento son más largos para las páginas web móviles que para las computadoras de escritorio. La mediana del tiempo total de subprocesos principales en el escritorio es de 849 ms, mientras que el móvil está en un número mayor: 2.436 ms.
```

File: src/content/es/2019/javascript.md [hunk 8]
```diff
@@ -0,0 +1,439 @@
...
+
+<figure>
+	<a href="/static/images/2019/01_JavaScript/js-processing-reddit.png">
+		<img src="/static/images/2019/01_JavaScript/js-processing-reddit.png" alt="Figura 4. Tiempos de procesamiento de JavaScript para Reddit.com" aria-labelledby="fig4-caption" aria-describedby="fig4-description" width="600">
+	</a>
+	<div id="fig4-description" class="visually-hidden">Gráfico de barras que muestra 3 dispositivos diferentes: en la parte superior, un Pixel 3, que tiene un tiempo de procesamiento pequeño tanto en el hilo principal como en el hilo worker de menos de 400 ms. Para un Moto G4 es de aproximadamente 900 ms en el subproceso principal y otros 300 ms en el subproceso del worker. Y la barra final es un Alcatel 1X 5059D con más de 2,000 ms en el subproceso principal y más de 500 ms en el subproceso del worker.</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig4-description" class="visually-hidden">Gráfico de barras que muestra 3 dispositivos diferentes: en la parte superior, un Pixel 3, que tiene un tiempo de procesamiento pequeño tanto en el hilo principal como en el hilo worker de menos de 400 ms. Para un Moto G4 es de aproximadamente 900 ms en el subproceso principal y otros 300 ms en el subproceso del worker. Y la barra final es un Alcatel 1X 5059D con más de 2.000 ms en el subproceso principal y más de 500 ms en el subproceso del worker.</div>
```

File: src/content/es/2019/javascript.md [hunk 9]
```diff
@@ -0,0 +1,439 @@
...
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig18.png">
+      <img src="/static/images/2019/01_JavaScript/fig18.png" alt="Figura 18. Porcentaje de sitios que usan mapas fuente." aria-labelledby="fig18-caption" aria-describedby="fig18-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=906754154&format=interactive">
+   </a>
+	<div id="fig18-description" class="visually-hidden">Gráfico de barras que muestra el 18% de los sitios de escritorio y el 17% de los sitios móviles utilizan mapas fuente.</div>
+	<figcaption id="fig18-caption">Figura 18. Porcentaje de sitios que usan mapas fuente.</figcaption>
+</figure>
+
+Para las páginas de escritorio y móviles, los resultados son casi los mismos. 17-18% incluye un mapa fuente para al menos un script en la página (detectado como un script de contenido de origen con  `sourceMappingURL`).
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Para las páginas de escritorio y móviles, los resultados son casi los mismos. Un 17-18% incluye un mapa fuente para al menos un script en la página (detectado como un script de contenido de origen con `sourceMappingURL`).
```

File: src/content/es/2019/javascript.md [hunk 10]
```diff
@@ -0,0 +1,439 @@
...
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig17.png">
+      <img src="/static/images/2019/01_JavaScript/fig17.png" alt="Figura 17. Uso de nuevas API de JavaScript." aria-labelledby="fig17-caption" aria-describedby="fig17-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=594315296&format=interactive">
+   </a>
+	<div id="fig17-description" class="visually-hidden">Gráfico de barras que muestra el 25.5% / 36.2% de los sitios en computadoras de escritorio y dispositivos móviles usa WeakMap, 6.1% / 17.2% usa WeakSet, 3.9% / 14.0% usa Intl, 3.9% / 4.4% usa Proxy, 0.4% / 0.4% usa Atomics, y 0.2% / 0.2% usan SharedArrayBuffer.</div>
+	<figcaption id="fig17-caption">Figura 17. Uso de nuevas API de JavaScript.</figcaption>
+</figure>
+
+Atomics (0.38%) y SharedArrayBuffer (0.20%) son apenas visibles en este gráfico ya que se usan en tan pocas páginas.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Atomics (0,38%) y SharedArrayBuffer (0,20%) son apenas visibles en este gráfico ya que se usan en tan pocas páginas.
```

File: src/content/es/2019/javascript.md [hunk 11]
```diff
@@ -0,0 +1,439 @@
...
+- [SharedArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+- [WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
+- [WeakSet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet)
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig17.png">
+      <img src="/static/images/2019/01_JavaScript/fig17.png" alt="Figura 17. Uso de nuevas API de JavaScript." aria-labelledby="fig17-caption" aria-describedby="fig17-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=594315296&format=interactive">
+   </a>
+	<div id="fig17-description" class="visually-hidden">Gráfico de barras que muestra el 25.5% / 36.2% de los sitios en computadoras de escritorio y dispositivos móviles usa WeakMap, 6.1% / 17.2% usa WeakSet, 3.9% / 14.0% usa Intl, 3.9% / 4.4% usa Proxy, 0.4% / 0.4% usa Atomics, y 0.2% / 0.2% usan SharedArrayBuffer.</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig17-description" class="visually-hidden">Gráfico de barras que muestra el 25,5% / 36,2% de los sitios en computadoras de escritorio y dispositivos móviles usa WeakMap, 6,1% / 17,2% usa WeakSet, 3,9% / 14,0% usa Intl, 3,9% / 4,4% usa Proxy, 0,4% / 0,4% usa Atomics, y 0,2% / 0,2% usan SharedArrayBuffer.</div>
```

File: src/content/es/2019/javascript.md [hunk 12]
```diff
@@ -0,0 +1,439 @@
...
+
+Para todos los sitios medidos en HTTP Archive, el 14.33% de los sitios de computadoras de escritorio y el 14.84% de los sitios en dispositivos móviles usan `<link rel="preload">` para los scripts en su página.
+
+Para _prefetch_:
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig16.png">
+      <img src="/static/images/2019/01_JavaScript/fig16.png" alt="Figura 16. Porcentaje de sitios que usan rel=prefetch para scripts." aria-labelledby="fig16-caption" aria-describedby="fig16-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=547807937&format=interactive">
+   </a>
+	<div id="fig16-description" class="visually-hidden">Gráfico de barras que muestra el 0.08% de los sitios en computadoras de escritorio usan 'rel=prefetch' y el 0.08% de los sitios en dispositivos móviles.</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig16-description" class="visually-hidden">Gráfico de barras que muestra el 0,08% de los sitios en computadoras de escritorio usan 'rel=prefetch' y el 0,08% de los sitios en dispositivos móviles.</div>
```

File: src/content/es/2019/javascript.md [hunk 13]
```diff
@@ -0,0 +1,439 @@
...
+	<figcaption id="fig16-caption">Figura 16. Porcentaje de sitios que usan rel=prefetch para scripts.</figcaption>
+</figure>
+
+Tanto para dispositivos móviles como para computadoras de escritorio, el 0.08% de las páginas aprovechan la captación previa para cualquiera de sus scripts.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Tanto para dispositivos móviles como para computadoras de escritorio, el 0,08% de las páginas aprovechan la captación previa para cualquiera de sus scripts.
```

File: src/content/es/2019/javascript.md [hunk 14]
```diff
@@ -0,0 +1,439 @@
...
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig14.png">
+      <img src="/static/images/2019/01_JavaScript/fig14.png" alt="Figura 14. Porcentaje de sitios que usan nomodule." aria-labelledby="fig14-caption" aria-describedby="fig14-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=781034243&format=interactive">
+   </a>
+	<div id="fig14-description" class="visually-hidden">Gráfico de barras que muestra el 0,8% de los sitios en computadoras de escritorio usan 'nomodule' y el 0,5% de los sitios en dispositivos móviles.</div>
+	<figcaption id="fig14-caption">Figura 14. Porcentaje de sitios que usan nomodule.</figcaption>
+</figure>
+
+Del mismo modo, muy pocos sitios (0.50% - 0.80%) usan el atributo `nomodule` para cualquier script.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Del mismo modo, muy pocos sitios (0,50% - 0,80%) usan el atributo `nomodule` para cualquier script.
```

File: src/content/es/2019/javascript.md [hunk 15]
```diff
@@ -0,0 +1,439 @@
...
+   </a>
+	<div id="fig13-description" class="visually-hidden">Gráfico de barras que muestra el 0.6% de los sitios en computadoras de escritorio usan 'type = module' y el 0.8% de los sitios en dispositivos móviles.</div>
+	<figcaption id="fig13-caption">Figura 13. Porcentaje de sitios que utilizan type=module.</figcaption>
+</figure>
+
+El soporte a nivel de navegador para módulos todavía es relativamente nuevo, y los números aquí muestran que muy pocos sitios usan actualmente `type="module"` para sus scripts. Muchos sitios todavía dependen de cargadores de módulos (2.37% de todos los sitios de escritorio usan [RequireJS](https://github.com/requirejs/requirejs) por ejemplo) y _bundlers_ ([webpack](https://webpack.js.org/) por ejemplo) para definir módulos dentro de su código fuente.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
El soporte a nivel de navegador para módulos todavía es relativamente nuevo, y los números aquí muestran que muy pocos sitios usan actualmente `type="module"` para sus scripts. Muchos sitios todavía dependen de cargadores de módulos (2,37% de todos los sitios de escritorio usan [RequireJS](https://github.com/requirejs/requirejs) por ejemplo) y _bundlers_ ([webpack](https://webpack.js.org/) por ejemplo) para definir módulos dentro de su código fuente.
```

File: src/content/es/2019/javascript.md [hunk 16]
```diff
@@ -0,0 +1,439 @@
...
+<script type="module" src="main.mjs"></script>
+```
+
+**¿Cuántos sitios usan `type ="module"` para los scripts en su página?**
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig13.png">
+      <img src="/static/images/2019/01_JavaScript/fig13.png" alt="Figura 13. Porcentaje de sitios que utilizan type=module." aria-labelledby="fig13-caption" aria-describedby="fig13-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1409239029&format=interactive">
+   </a>
+	<div id="fig13-description" class="visually-hidden">Gráfico de barras que muestra el 0.6% de los sitios en computadoras de escritorio usan 'type = module' y el 0.8% de los sitios en dispositivos móviles.</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig13-description" class="visually-hidden">Gráfico de barras que muestra el 0,6% de los sitios en computadoras de escritorio usan 'type=module' y el 0,8% de los sitios en dispositivos móviles.</div>
```

File: src/content/es/2019/javascript.md [hunk 17]
```diff
@@ -0,0 +1,439 @@
...
+
+Aunque este análisis es interesante, es importante tener en cuenta que estos resultados se basan en una biblioteca de detección de terceros - [Wappalyzer](https://www.wappalyzer.com). Todos estos números de uso dependen de la precisión de cada uno de los [mecanismos de detección](https://github.com/AliasIO/Wappalyzer/blob/master/src/apps.json).
+
+## Carga diferenciada
+
+Los [módulos JavaScript](https://v8.dev/features/modules), o ES modules, son soportados en [en todos los navegadores principales](https://caniuse.com/#feat=es6-module). Los módulos proporcionan la capacidad de crear scripts que pueden importar y exportar desde otros módulos. Esto permite a cualquier persona construir sus aplicaciones diseñadas en un patrón de módulo, importando y exportando donde sea necesario, sin depender de cargadores de módulos de terceros.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Los [módulos JavaScript](https://v8.dev/features/modules), o ES modules, son soportados en [todos los navegadores principales](https://caniuse.com/#feat=es6-module). Los módulos proporcionan la capacidad de crear scripts que pueden importar y exportar desde otros módulos. Esto permite a cualquier persona construir sus aplicaciones diseñadas en un patrón de módulo, importando y exportando donde sea necesario, sin depender de cargadores de módulos de terceros.
```

File: src/content/es/2019/javascript.md [hunk 18]
```diff
@@ -0,0 +1,439 @@
...
+
+**¿Cuántos sitios usan este tipo de frameworks?**
+
+<figure>
+   <a href="/static/images/2019/01_JavaScript/fig12.png">
+      <img src="/static/images/2019/01_JavaScript/fig12.png" alt="Figura 12. Los frameworks más utilizados en el escritorio." aria-labelledby="fig12-caption" aria-describedby="fig12-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1699359221&format=interactive">
+   </a>
+	<div id="fig12-description" class="visually-hidden">Gráfico de barras que muestra que el 4.6% de los sitios usan React, 2.0% AngiularJS, 1.8% Backbone.js, 0.8% Vue.js, 0.4% Knockout.js, 0.3% Zone.js, 0.3% Angular, 0.1% AMP, 0.1% Ember. js.</div>
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
	<div id="fig12-description" class="visually-hidden">Gráfico de barras que muestra que el 4,6% de los sitios usan React, 2,0% AngiularJS, 1,8% Backbone.js, 0,8% Vue.js, 0,4% Knockout.js, 0,3% Zone.js, 0,3% Angular, 0,1% AMP, 0,1% Ember. js.</div>
```

File: src/content/es/2019/javascript.md [hunk 19]
```diff
@@ -0,0 +1,439 @@
...
+
+Hay varias razones posibles:
+
+- [WordPress](https://wordpress.org/), que se usa en más del 30% de los sitios, incluye jQuery por defecto.
+- Cambiar de jQuery a una biblioteca más nueva del lado del cliente puede llevar tiempo dependiendo de qué tan grande sea una aplicación, y muchos sitios pueden consistir en jQuery y además bibliotecas más nuevas del lado del cliente.
+
+Otras bibliotecas JavaScript más utilizadas incluyen variantes de jQuery (jQuery migrate, jQuery UI), [Modernizr](https://modernizr.com/), [Moment.js](https://momentjs.com/), [Underscore.js](https://underscorejs.org/) y así.
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]:
```suggestion
Otras bibliotecas JavaScript más utilizadas incluyen variantes de jQuery (jQuery migrate, jQuery UI), [Modernizr](https://modernizr.com/), [Moment.js](https://momentjs.com/), [Underscore.js](https://underscorejs.org/), etc.
```

File: src/content/es/2019/javascript.md [hunk 20]
```diff
@@ -0,0 +1,439 @@
...
+          <td>Dojo</td>
+          <td class="numeric">0.70%</td>
+          <td class="numeric">0.62%</td>
+        </tr>
+      </tbody>
+    </table>
+	<figcaption>Figura 11. Principales bibliotecas de JavaScript en computadoras de escritorio y dispositivos móviles.</figcaption>
+</figure>
+
+[jQuery](https://jquery.com/), la biblioteca JavaScript más popular jamás creada, se utiliza en el 85.03% de las páginas de escritorio y el 83.46% de las páginas móviles. El advenimiento de muchas API y métodos del navegador, tales como [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) y [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector), estandarizaron gran parte de la funcionalidad proporcionada por la biblioteca en una forma nativa. Aunque la popularidad de jQuery puede parecer estar disminuyendo, ¿por qué todavía se usa en la gran mayoría de la web?
```
Comment (JMPerez) [inline src/content/es/2019/javascript.md]: Removing `en-US` from MDN pages that have an equivalent page available in Spanish.

```suggestion
[jQuery](https://jquery.com/), la biblioteca JavaScript más popular jamás creada, se utiliza en el 85.03% de las páginas de escritorio y el 83.46% de las páginas móviles. El advenimiento de muchas API y métodos del navegador, tales como [Fetch](https://developer.mozilla.org/docs/Web/API/Fetch_API) y [querySelector](https://developer.mozilla.org/docs/Web/API/Document/querySelector), estandarizaron gran parte de la funcionalidad proporcionada por la biblioteca en una forma nativa. Aunque la popularidad de jQuery puede parecer estar disminuyendo, ¿por qué todavía se usa en la gran mayoría de la web?
```

**Comment by rviscomi** [comment]
@c-torres can you also run `npm run generate` to create the HTML version of the chapter?

**Comment by bazzadp** [comment]
> @c-torres can you also run `npm run generate` to create the HTML version of the chapter?

Don't think we can do this as don't have all the HTML templates translated for Spanish yet. So shall we just merge as is?

**Comment by rviscomi** [comment]
We can either run the script now as part of this PR, or the next person to run it will pick up these changes and it'll be included in their PR. I think it's best to keep all relevant artifacts in the same PR.

**Comment by rviscomi** [comment]
Disregard, @bazzadp was right. :)

**Pull request merged**